runtime/compilers/rillv1/parse_alert.go

```diff
@@ -25,8 +26,14 @@ type AlertYAML struct {
 		Limit         uint   `yaml:"limit"`
 		CheckUnclosed bool   `yaml:"check_unclosed"`
 	} `yaml:"intervals"`
-	Timeout string `yaml:"timeout"`
-	Query   struct {
+	Timeout string         `yaml:"timeout"`
+	Data    map[string]any `yaml:"data"` // will be parsed as DataYAML, is defined as raw map to save the original content
```

Comment:
We should use `DataYAML` here as in the other resources that use it. We should be able to avoid needing to capture the raw contents of it.